pom.xml

```diff
@@ -35,6 +35,7 @@
         <artifacts.folder>${resources.folder}/artifacts</artifacts.folder>
         <testng.version>6.1.1</testng.version>
         <mockito.version>1.10.19</mockito.version>
+        <java.agent.infra.version>[1.0.0-SNAPSHOT],[1.0,)</java.agent.infra.version>
```

Comment:
Are you sure that's the version we need?

Reply:
yup, its the same as all our java projects